types/esri-leaflet-geocoder/index.d.ts

```diff
@@ -50,7 +50,7 @@ declare module 'leaflet' {
         }
 
         class Geocode extends Task {
-            constructor(options?: TaskOptions | Service);
+            constructor(options?: ServiceOptions | Service);
```

Comment:
It does seem a little unfortunate that the new version of `esri-leaflet` is not backward compatible seemingly only because of this one type name.

Reply:
Probably still worthwhile to get this up to date, since 3.0 is 3 years old now.